data/en/lgbtq.yml

```diff
@@ -147,13 +156,15 @@
     - genetically female
 - type: simple
   source: https://www.glaad.org/reference/transgender
+  note: A term created and used by far-right extremists to oppose nondiscrimination laws that protect transgender people.
   considerate:
     - non-discrimination law
     - non-discrimination ordinance
   inconsiderate:
     - bathroom bill
 - type: simple
   source: http://www.isna.org/node/979
+  note: These terms are stigmatizing to patients and their families. Intersex status is more complicated than the mere presence or absence of certain gonadal tissues.
```

Comment:
```suggestion
  note: These terms are stigmatizing to patients and their families because intersex status is more complicated than the mere presence or absence of certain gonadal tissues
```